app/code/Magento/Checkout/CustomerData/Cart.php

```diff
@@ -90,8 +91,13 @@ public function getSectionData()
     {
         $totals = $this->getQuote()->getTotals();
         $subtotalAmount = $totals['subtotal']->getValue();
+        $summaryCount = $this->getSummaryCount();
+        if ($summaryCount>0) {
+            $store = ObjectManager::getInstance()->get(\Magento\Framework\Locale\Resolver::class);
```

Comment:
All the dependencies should be initialized in the `__constructor` method. Please add new private property called `localeResolver`.
Please also use the `\Magento\Framework\Locale\ResolverInterface` instead of `\Magento\Framework\Locale\Resolver`.